src/audio/buffers/ring_buffer.c

```diff
@@ -19,14 +19,14 @@ DECLARE_TR_CTX(ring_buffer_tr, SOF_UUID(ring_buffer_uuid), LOG_LEVEL_INFO);
 
 static inline struct ring_buffer *ring_buffer_from_sink(struct sof_sink *sink)
```

Comment:
I'd still add the "audio: " prefix to keep up with the convention to have a module prefix in commits.